hdf5.spec

```
@@ -1,29 +1,24 @@
 %global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
 %{!?_fmoddir:%global _fmoddir %{_libdir}/gfortran/modules}
 
-%global daos_major 0
-%global hdf5_commit fa40c6c59af5d9aabd4b478cd02f8a9f7ebf7922
-%define hdf5_sha .gfa40c6c59a
+%global daos_major 1
 
 # Patch version?
 %global snaprel %{nil}
-# HDF5 versions
-%global major 1.12
-%global minor 0
 
 # NOTE:  Try not to release new versions to released versions of Fedora
 # You need to recompile all users of HDF5 for each version change
 Name: hdf5
-Version: %{major}.%{minor}
-Release: 5%{hdf5_sha}%{?dist}
+Version: %{hdf5_major}_%{hdf5_minor}
+Release: %{hdf5_release}%{?relval}%{?dist}
 Summary: A general purpose library and file format for storing scientific data
 License: BSD
-URL: https://portal.hdfgroup.org/display/HDF5/HDF5
+URL: https://github.com/HDFGroup/hdf5
 
-Source0: https://github.com/HDFGroup/hdf5/archive/%{hdf5_commit}.tar.gz
+Source0: https://github.com/HDFGroup/hdf5/archive/%{name}-%{name}-%{version}-%{hdf5_release}.tar.gz
```

```suggestion
Source0: https://github.com/HDFGroup/hdf5/archive/%{name}-%{hdf5_major}_%{hdf5_minor}-%{hdf5_release}.tar.gz
```
is closer to what you need, modulo the confusion about `.` and `_` in the `%{hdf_minor]`.  It seems like you need to separate out the bugfix portion from the minor into a separate `%{hdf5_bugfix}`.